Applications/Terminal/Terminal.cpp

```diff
@@ -135,6 +134,36 @@ static inline Color lookup_color(unsigned color)
     return Color::from_rgb(xterm_colors[color]);
 }
 
+void Terminal::escape$l(const ParamVector& params)
+{
+    int mode = 2;
+    if (params.size() > 0) {
+        mode = params[0];
+    }
+    if (!m_question_param) {
+        switch (mode) {
+            // FIXME: implement *something* for this
+        default:
+            unimplemented_escape();
+            break;
+        }
+    } else {
+        switch (mode) {
+        case 25:
+            // Hide cursor command, but doesn't need to be run (for now, because
+            // we don't do inverse control codes anyways)
+            break;
+        default:
+            break;
+        }
+    }
+}
+
+void Terminal::escape$h(const ParamVector& params)
```

Comment:
IIRC, h and l are basically the same thing. 'h' is to set modes, 'l' is to remove them. It might make sense to combine the two function bodies, and have a boolean 'should_set' or something parameter.

Reply:
Same here